management/proto/management.proto

```diff
@@ -31,13 +31,19 @@ message EncryptedMessage {
   bytes body = 2;
 }
 
-message SyncRequest {
-
-}
+message SyncRequest {}
 
+// SyncResponse represents a state that should be applied to the local peer (e.g. Wiretrustee servers config as well as local peer and remote peers configs)
 message SyncResponse {
-  // A list of peers available
-  repeated string peers = 1;
+  // Global config
```

Comment:
Yes, I wanted to tell you that we need this config. Strange a little bit that it's in a SyncResponse, not in Register but it's ok too.